### PR TITLE
EIP-1191: Change status to Last Call

### PR DESCRIPTION
**Pull request by juli**
Change status to Last Call and add reference to web3.js and ethereumjs-util implementations.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 1191 state from Draft to Last Call

File: EIPS/eip-1191.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 eip: 1191 
 title: Add chain id to mixed-case checksum address encoding 
 author: Juliano Rizzo (@juli)
-status: Draft
+status: Last Call
```
Comment (axic) [inline EIPS/eip-1191.md]: This status needs a `review-period-end` field.

Comment (axic) [reply EIPS/eip-1191.md]: This is still missing.

**Comment by axic** [comment]
Please make sure to advertise this and gather sufficient agreement from adopters.

**Pull request merged**